srcpkgs/chromium/template

```diff
@@ -34,15 +35,15 @@ makedepends="libpng-devel gtk+-devel gtk+3-devel nss-devel pciutils-devel
  libjpeg-turbo-devel libevent-devel json-c-devel harfbuzz-devel
  minizip-devel jsoncpp-devel zlib-devel libcap-devel libXdamage-devel
  re2-devel fontconfig-devel freetype-devel opus-devel
- ffmpeg-devel libva-devel python-setuptools xcb-proto
+ ffmpeg-devel libva-devel python-setuptools xcb-proto libpipewire0.2-devel
  $(vopt_if sndio sndio-devel)"
 depends="libexif hwids desktop-file-utils hicolor-icon-theme xdg-utils"
```

Comment:
Add `$(vopt_if pipewire libpipewire0.2)`

Reply:
I believe @ifreund suggests that pipewire is not a hard dependency, so we can probably leave it out (compile support for it, but not make it required).

I can confirm that I can run chromium without it, so that should be OK.

Reply:
We should really add optional dependencies into `xbps`.